src/libs/utils.js

```diff
@@ -256,7 +256,7 @@ export const outputCommaSeperatedElectionStatuses = (elections) => {
     elections.map((e) => processElectionStatus(e, e.votes, false))
   ).filter((status) => !isEmpty(status));
   if (isEmpty(statuses)) {
-    return 'Unreviewed';
+    return 'Submitted';
```

Comment:
I'm sure you tested this out locally, but I don't see how this text is being passed back to the display code. The only usage of this method that I see is in `CancelCollectionButton`, but that component appears to be unused.

To show that this works to fix the UI, in addition to a unit test, it would be good to have a UI test that verifies at the text of a rendered component.

Reply:
Right .. I'm pretty sure this PR isn't doing what the ticket describes. Running this locally, I still see `Unreviewed` in the researcher DAR table